README.rst

```diff
@@ -254,7 +254,7 @@ _____________
 
 .. code-block:: bash
 
-    docker run -d -p 5672:5672 -p 15672:15672 rabbitmq:3-management
+    docker run -d -p 5671:5671 -p 5672:5672 -p 15671:15671 -p 15672:15672 mosquito/aiormq-rabbitmq
```

Comment:
Nice fix. Don't you want to break this into escaped multiline string?

Reply:
I do not want to. But I will if you insist.

Reply:
I don't insist. I just wanted to see if anyone else wants to see this in a few lines.